homework_5/task_3.py

```diff
@@ -0,0 +1,15 @@
+# Создайте функцию генератор чисел Фибоначчи (см. Википедию)
+
+num = int(input('Введите номер элемента Фиббоначи: '))
+
+
+def fibo_calc(n):
+    n = int(n) - 2
+    a = b = 1
+    while n:
+        a, b = b, a + b
+        n -= 1
+    yield b
```

Comment:
Зачем вообще что-то передавать функции? Вот как должен выглядеть такой код
```suggestion
def fibo_calc():
    a, b = 0, 1
    while True:
        yield a
        a, b = b, a + b
```
Вот и все

Reply:
Спасибо.